### PR TITLE
refactored prelogin and preloginoutages

### PR DESCRIPTION
**Pull request by EujinGG**

### CODE REVIEW

File: src/test/java/demo/tests/scp/PreLoginOutagesTests.java [hunk 1]
```diff
@@ -16,10 +17,9 @@ public class PreLoginOutagesTests extends TestRunner {
 	private static final Logger log = LogManager.getLogger(PreLoginOutagesTests.class);
 	private PreLoginOutagesSteps preLoginOutagesSteps;
 	public static PropertiesUtil loginTextProp;
-	String accountType = null;
 
-	@FrameworkAnnotations(author = { "Eujin" }, category = { CategoryType.SANITY })
-	@Test(priority = 1, description = "To verify the pre-login outages page objects.", groups = { "sanity" })
```
Comment (PriyaAwas) [inline src/test/java/demo/tests/scp/PreLoginOutagesTests.java:22]: Add one more category from which module it belongs

File: src/test/java/demo/tests/scp/PreLoginOutagesTests.java [hunk 2]
```diff
@@ -28,12 +28,12 @@ public void verifyPreLoginOutagesPageObjects() {
...
 		// Assert all the soft verification.
 		softAssert.assertAll();
+		ExtentLogger.logInfo("Test Case execution for - verifyPreLoginOutagesPageObjects - is Completed.");
 		log.info("Test Case execution for - verifyPreLoginOutagesPageObjects - is Completed.");
 	}
 
-	@FrameworkAnnotations(author = { "Eujin" }, category = { CategoryType.SANITY })
-	@Test(priority = 2, description = "To verify the toggle icon to switch the view between map view and list view.", groups = {
-			"sanity" })
+	@FrameworkAnnotations(author = { "Eujin" }, category = { CategoryType.SMOKE })
```
Comment (PriyaAwas) [inline src/test/java/demo/tests/scp/PreLoginOutagesTests.java:35]: all the test cases will not come under the smoke only smoke suite one others will come under the sanity + module

File: src/test/java/demo/tests/scp/PreLoginOutagesTests.java (hunk 1 above)
Comment (EujinGG) [reply src/test/java/demo/tests/scp/PreLoginOutagesTests.java:22]: ok

**Review by PriyaAwas — CHANGES_REQUESTED** [review]
Update the review comments

File: src/test/java/demo/tests/scp/PreLoginOutagesTests.java (hunk 1 above)
Comment (EujinGG) [reply src/test/java/demo/tests/scp/PreLoginOutagesTests.java:22]: have made this changes

File: src/test/java/demo/tests/scp/PreLoginOutagesTests.java (hunk 2 above)
Comment (EujinGG) [reply src/test/java/demo/tests/scp/PreLoginOutagesTests.java]: made these changes

**Comment by EujinGG** [comment]
I have made the changes that has been requested

**Pull request merged**